### ISSUE TITLE
Model2 Native Crosshairs no longer draw in the correct position since v10.

### ISSUE THREAD

**Issue by adamgp** [argonlefou/DemulShooter#25]
Hi, I believe I may have noticed an edge case and difference between pre v10 and the latest DemulShooter (.13) when it comes to Model2 and the native emulator crosshairs (without the DemulShooter LUA scripts installed).

Using dual Aimtraks and following the (updated) documentation with UseRawInput=0, the (single red) crosshair correctly follows where the guns are actually aiming (and where the bullets decals are rendered upon firing). However, this single red crosshair will flick between the two gun positions, much like a Windows cursor will do when using dual AimTrak absolute mice pointers on the desktop.

In an attempt to restore dual crosshairs (red and blue) using the old method of UseRawInput=1, both the cross hairs are shown, but they are stuck in the top left corner. The aiming of the guns is intact as the bullet decals show and the game is fully playable but the individual crosshairs no longer represent where the gun is actually aiming.

I then tried to resolve but setting RawDevP1 and RawDevP2 to various index numbers hoping the emulator would detect the mouse position and correctly align the crosshairs but this failed as the crosshairs jumped from corner to corner.

So to summarise, whilst DemulShooter operates perfectly fine with either UseRawInput set as 0 or 1, I cannot get both crosshairs to show accurately at the same time - whereas, pre v10 this worked as intended.

Any advice would be greatly appreciated, thank you.

**Comment by adamgp** [comment]
I have achieved dual crosshair support by setting DrawCross=0 and using your custom LUA scripts.
I would suggest the Wiki needs an update to mention that when using DrawCross=1 - support only exists for one shared crosshair just to be clear.

Thank you.
